### DISCUSSION TITLE
Welcome to Idena Discussions!

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-go#636] [General]
## 👋 Welcome!
  We're using Discussions as a place to share and discuss Idena improvements proposals.

**Comment by thelemoe** [comment]
I propose changing the flip reporting threshold from > 50% to >= 50%, because reports are already under-utilized and this would make it easier to penalize people submitting flips that are poor quality or rule violations (not AI-resistant).

**Reply by midenaio** [reply]
It may help in some cases. Would it be better to incentive people to report the flips more thoroughly?

**Comment by TravCrypto** [comment]
This is my first GitHub comment.

**Reply by TravCrypto** [reply]
This is my first reply to a GitHub comment.
